lib/cartopy/mpl/geoaxes.py

```diff
@@ -988,9 +989,13 @@ def stock_img(self, name='ne_shaded'):
         """
         Add a standard image to the map.
 
-        Currently, the only (and default) option is a downsampled version of
-        the Natural Earth shaded relief raster.
+        Currently, there are 2 options:
 
+        1. 'ne_shaded'(default) a downsampled version of the Natural Earth
+           shaded relief raster.
+        2. 'etopo' a downsampled version of global relief model of Earth's
+           surface that integrates land topography and ocean bathymetry. This
+           option is the same as the etopo from Basemap.
```

Comment:
Can you put in a link to each of these for the documentation so people can see what they are going to get?